Tests/AttachmentStorageTests.swift

```diff
@@ -0,0 +1,60 @@
+import XCTest
+
+import Nimble
+import Quick
+@testable import Backtrace
+
+final class AttachmentStorageTests: QuickSpec {
+    
+    override func spec() {
+        describe("AttachmentStorage") {
+            it("can save attachments as a plist") {
+                var crashAttachments = Attachments()
+                guard let fileUrl = try? self.createAFile() else {
```

Comment:
Do we create physical file on hard drive?

Reply:
Yes, we are creating a physical file on the hard drive here.

Reply:
Are you sure we should create file in unit tests project? Unit tests shouldn't create files on the hard drive. @krml19 what do you think?

Reply:
Yes, I agree. It'd be definitely better to use a mock

Reply:
@krml19 @konraddysput Even if I mock my dummy file here, `AttachmentsStorage.store` function would still write a plist containing the attachment paths, just like `AttributesStorage.store` function writes a plist containing attributes. Is it still worth it to create a mock filesystem here for my dummy file? We wouldn't fully eliminate file system writes/dependencies anyway because of `AttachmentsStorage.store` internal details.

Reply:
I think it can be resolved by the dependency injection technique. In other words, one can swap and inject all the required external dependencies.

Reply:
@krml19 @konraddysput I have injected all filesystem dependencies, please take another look!